metrics_api/.rubocop.yml

```diff
@@ -15,8 +16,14 @@ Metrics/CyclomaticComplexity:
   Max: 20
 Metrics/ParameterLists:
   Enabled: false
+
+Naming/VariableNumber:
+  Enabled: false
```

Comment:
I happy to revert any of these style changes, just let me know

nit: For this one, I think that naming variables with `something_1` and `something_2` in tests is easier to read